src/pages/Courses/utils/filterReducer.js

```diff
@@ -0,0 +1,21 @@
+export const filterReducer = (state, action) => {
```

Comment:
Even if it was intentional, here's what I noticed:
1. Clear All button is not resetting all the filter states. 
2. When moving to another page and returning to the products page, the filter state is getting reset.
3. Couldn't find the values of the rating being rendered on the Product Card.

![image](https://user-images.githubusercontent.com/91944618/160283630-74e2b742-b14e-43bc-9502-bc1b0dc72b9a.png)